### ISSUE TITLE
Change math rendering in the doc

### ISSUE THREAD

**Issue by m-balesdent** [openturns/openturns#2147]
Hello,

Currently, LaTeX notations in the doc are converted into pictures. An alternative way to enhance readability could be to use mathjax extension of sphinx.

It could be done by adding `extensions.add('sphinx.ext.mathjax')` in python/doc/conf.py.in and removing lines dealing with sphinx.ext.imgmath.

![image](https://user-images.githubusercontent.com/26813843/193278563-0ac0fd65-07ea-4503-9675-94321fbc9214.png)

Mathieu

**Comment by jschueller** [comment]
we already been there and switched back to imgmath, mathjax cdns are unreliable and can cometimes be blocked

**Comment by m-balesdent** [comment]
This is a pity, because it provides very interesting potentialities in terms of directly extracting latex formulas by right clicking .

![image](https://user-images.githubusercontent.com/26813843/193288510-7e66aceb-678f-4b5a-abaf-de2279a4e1be.png)

**Comment by jschueller** [comment]
maybe that feature could be added to imgmath as well (I dont know how though), because it knows the latex source too

**Comment by jschueller** [comment]
an advantage of mathjax is that its faster to build & upload (no image files), but a downside is that latex syntax is not checked at compilation time, that's a no-go for us

**Comment by m-balesdent** [comment]
would it be possible to check latex syntax with dvipng and compile with mathjax?

**Comment by jschueller** [comment]
mathjax does not compile anything, but no I think you have to choose

**Comment by m-balesdent** [comment]
so to use mathjax, it will be mandatory to have  another plug in to check latex syntax...

**Comment by m-balesdent** [comment]
@jschueller, I have found on this [page](https://docs.mathjax.org/en/latest/options/input/tex.html) that there is Tex input processor options on MathJax that may allow to return errors in the TeX that it is processing?

**Comment by jschueller** [comment]
I'd rather change the math font: https://github.com/sphinx-doc/sphinx/issues/5301

**Comment by m-balesdent** [comment]
I think this discussion is outdated.
